### PR TITLE
Add more improvements about l10n

### PR DESCRIPTION
**Pull request by macagua**
- Add Transifex.net service integration to manage the translation process.
- Add Spanish translation
- Update README and CONTRIBUTORS files

### CODE REVIEW

**Comment by macagua** [comment]
Hi @gbastien and @sgeulette 

Can you review this pull request, if all ok can merge it?

Thank you

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+#. Default: "Item state changed."
+#: ./browser/views.py:502
+msgid "%s_done_descr"
+msgstr "Estado del elemento cambiado."
+
+#: ./browser/registry.py:16
+msgid "Define for each object transition the optional corresponding view"
+msgstr ""
+"Definir para cada transición de objeto la vista correspondiente opcional"
```
Comment (gbastien) [inline src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: Hi @macagua 

this seems a typo error?

Thank you for this PR,

Gauthier

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+msgid "Transitions to confirm"
+msgstr "Transiciones a confirmar"
+
+#: ./browser/actions_panel_add_content.pt:6
+msgid "Add an element"
+msgstr "Añadir un elemento"
+
+#: ./browser/jsvariables.py:15
+msgid "delete_confirm_message"
+msgstr ""
```
Comment (gbastien) [inline src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: Same here?

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+"aplicación?"
+
+#: ./browser/actions_panel_history.pt
+msgid "history.gif_icon_title"
+msgstr "Historial del elemento"
+
+#: ./browser/actions_panel_history.pt
+msgid "history_last_event_has_comment.gif_icon_title"
+msgstr ""
+"Historial de elemento (se ingresó un comentario durante el último evento)"
```
Comment (gbastien) [inline src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: ?

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+#: ./browser/transitions.pt:21
+msgid "transition_comment"
+msgstr "Comentario"
+
+#. Default: "Confirm workflow transition ${transition_name}"
+#: ./browser/transitions.pt:20
+msgid "workflow_confirm"
+msgstr ""
+"¿Estás seguro de que deseas confirmar la transición del flujo de trabajo "
```
Comment (gbastien) [inline src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: ?

**Review by gbastien — COMMENTED** [review]
Hi @macagua 

thank you for this, just some remarks, maybe that work?

I hope everything is going well for you in Venezuela with all this Covid19 thing...

Thank you,

Gauthier

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po (hunk 1 above)
Comment (macagua) [reply src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: Fixed on commit [37966a0]

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po (hunk 2 above)
Comment (macagua) [reply src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: Fixed on commit [37966a0]

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po (hunk 3 above)
Comment (macagua) [reply src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: Fixed on commit [37966a0]

File: src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po (hunk 4 above)
Comment (macagua) [reply src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po]: Fixed on commit [37966a0]

**Comment by macagua** [comment]
Hi @gbastien 

I am here in Venezuela, for the moment I am very well, you know, I staying home, and take it back my contributions for imio projects.

I hope you are very well in Belgium, safe of Covid-19.

**Comment by gbastien** [comment]
@macagua 

merged, thank you for contribution.

See you soon, maybe at the next PloneConference in november in Belgium (Namur) ;-)

Gauthier

**Pull request merged**